packages/docusaurus-plugin-content-docs/src/docs.ts

```diff
@@ -50,6 +51,7 @@ type LastUpdateOptions = Pick<
 async function readLastUpdateData(
```

Comment:
This doesn't need to be async. We can make this sync so that the downstream can be sync as well.

Reply:
actually when replacing shelljs with execa I'd rather make all this async when executing shell commands. We shouldn't block nodejs while reading git lastupdate date, this may actually be a major bottleneck